contract_manager/src/base.ts

```diff
@@ -42,12 +54,30 @@ export abstract class Contract extends Storable {
    */
   abstract getBaseUpdateFee(): Promise<{ amount: string; denom?: string }>;
 
+  /**
+   * Returns the last governance sequence that was executed on this contract
+   * this number increases based on the sequence number of the governance messages
+   * that are executed on this contract
+   *
+   * This is used to determine which governance messages are stale and can not be executed
+   */
+  abstract getLastExecutedGovernanceSequence(): Promise<number>;
+
+  /**
+   * Returns the price feed for the given feed id
+   * @param feedId hex encoded feed id without 0x prefix
+   */
+  abstract getPriceFeed(feedId: string): Promise<PriceFeed | undefined>;
+
   /**
    * Executes the governance instruction contained in the VAA using the sender credentials
-   * @param sender based on the contract type, this can be a private key, a mnemonic, a wallet, etc.
+   * @param senderPrivateKey private key of the sender in hex format without 0x prefix
```

Comment:
So far, I'm seeing hex format as the more common input format. I think if the implementation needs the uint8 array or Buffer it can easily convert the privateKey via `Buffer.from`